cypress/support/index.js

```diff
@@ -18,3 +18,9 @@ import './commands'
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+  })
```

Comment:
I grabbed this straight from the Cypress docs. This is to avoid app errors failing the tests, e.g.:
Screenshot from 2022-01-12 15-19-16
I've added a tech debt issue: #131